roles/openshift_metrics/meta/main.yaml

```diff
@@ -1,17 +1,2 @@
----
```

Comment:
There should be more metadata other than just the dependencies here. Also, the `---` header should be retained.